src/plugins/ceramic-feed/server/content-types/ceramic-post/lifecycles.js

```diff
@@ -236,16 +259,29 @@ const LifecycleHooks = new class {
     try {
       return await callback(this.ceramic)
     } catch (e) {
-      throw new ApplicationError('Ceramic Network sync failed. Please try again later...')
+      console.log('Error:', e)
+      throw new ApplicationError(
+        'Ceramic Network sync failed. Please try again later...'
+      )
     }
   }
-}(strapi, schema, relations)
+})(strapi, schema, relations)
 
 module.exports = {
-  async beforeUpdate(event) {
-    return LifecycleHooks.onUpdate(event)
+  async afterUpdate(event) {
+    console.log('afterUpdate', event)
```

Comment:
destructurize to make references simpler (const { data } = event.params)

Reply:
done